idea/jvm-debugger/jvm-debugger-test/testData/stepping/stepOver/coroutines/sequenceNested2.ir.out

```diff
@@ -0,0 +1,14 @@
+LineBreakpoint created at sequenceNested2.kt:8
+Run Java
+Connected to the target VM
+sequenceNested2.kt:8
+sequenceNested2.kt:9
+sequenceNested2.kt:10
+sequenceNested2.kt:11
+sequenceNested2.kt:23
+sequenceNested2.kt:23
+sequenceNested2.kt:23
```

Comment:
The difference here is an extra step on line number 23. That is because bridges on JVM_IR consistently have line numbers. The `invoke` bridge for the `{ it.toList() }` lambda does not have line numbers on the current backend. The line number on JVM_IR leads to the extra step.

I proposed rebasing here as we have done that for other similar cases. However, we can attempt to add another special case to avoid the line number in these bridges if we think it matters.

Reply:
I'm OK with this behavior, as long as we log it into a YouTrack issue (just for the sake of having all differences in behavior reported in one place)

Reply:
Filed an issue here: https://youtrack.jetbrains.com/issue/KT-42696